tests/storage/storage.go

```diff
@@ -1131,39 +1123,10 @@ var _ = SIGDescribe("Storage", func() {
 			It("should successfully start with distinct device names", func() {
 
 				// Start the VirtualMachineInstance with two empty disks attached, one per bus
-				vmi = tests.NewRandomVMIWithEphemeralDisk(cd.ContainerDiskFor(cd.ContainerDiskAlpine))
-				vmi.Spec.Domain.Devices.Disks = append(vmi.Spec.Domain.Devices.Disks, virtv1.Disk{
-					Name: "emptydisk1",
-					DiskDevice: virtv1.DiskDevice{
-						Disk: &virtv1.DiskTarget{
-							Bus: v1.DiskBusSCSI,
-						},
-					},
-				})
-				vmi.Spec.Domain.Devices.Disks = append(vmi.Spec.Domain.Devices.Disks, virtv1.Disk{
-					Name: "emptydisk2",
-					DiskDevice: virtv1.DiskDevice{
-						Disk: &virtv1.DiskTarget{
-							Bus: v1.DiskBusSATA,
-						},
-					},
-				})
-				vmi.Spec.Volumes = append(vmi.Spec.Volumes, virtv1.Volume{
-					Name: "emptydisk1",
-					VolumeSource: virtv1.VolumeSource{
-						EmptyDisk: &virtv1.EmptyDiskSource{
-							Capacity: resource.MustParse("1Gi"),
-						},
-					},
-				})
-				vmi.Spec.Volumes = append(vmi.Spec.Volumes, virtv1.Volume{
-					Name: "emptydisk2",
-					VolumeSource: virtv1.VolumeSource{
-						EmptyDisk: &virtv1.EmptyDiskSource{
-							Capacity: resource.MustParse("1Gi"),
-						},
-					},
-				})
+				vmi = libvmi.NewAlpine(
+					libvmi.WithEmptyDisk("emptydisk1", "1Gi", v1.DiskBusSCSI),
+					libvmi.WithEmptyDisk("emptydisk2", "1Gi", v1.DiskBusSATA),
```

Comment:
:+1: 
this looks so good, when compared to previous one ;)

Reply:
Thanks :)

Reply:
I'd say that it's so good that it makes the comment above it redundant.